lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -47,6 +47,64 @@ def initialize(target)
           @target = target
         end
 
+        class << self
+          # TODO
+          #
+          def add_embed_frameworks_script_phase_to_target(native_target, script_path, input_paths, output_paths)
```

Comment:
These became helper class methods, if you'd like to me to move them to another file let me know. It is a bit weird they are under `class UserProjectIntegrator` and they are used for non user project integrated targets.